### ISSUE TITLE
Allow broker to start when no .conf files are found in include_dir

### ISSUE THREAD

**Issue by marleau** [eclipse/mosquitto#566]
I recently came across an issue where the broker wouldn't start if the include_dir did not contain a .conf file. In my use case, the folder may not always have a .conf file because extra configuration is not always needed.

Please allow the broker to start when no external configuration files are found in the include_dir.

**Comment by toast-uz** [comment]
I confirmed this issue.
On Windows, if the include_dir did not contain a .conf file, the broker saids:

```
Error: Unable to open include_dir 'testing'.
Error found at test_include.conf:1.
Error: Unable to open configuration file.
```

While no problem on the same situation on Linux.

**Comment by toast-uz** [comment]
I reconfirmed conf.c source file.

Mosquitto on Windows calls FindFirstFile Win32API to check existence of both of directory and *.conf files at the same time, while on Linux does opendir system call  to check existence of directory only.

**Comment by ralight** [comment]
This is now fixed in my local branch and will be released shortly as part of 1.5.3.
